src/lib/core/datetime/native-date-adapter.ts

```diff
@@ -48,13 +47,15 @@ function range<T>(length: number, valueFunction: (index: number) => T): T[] {
   return valuesArray;
 }
 
+/** InjectionToken for datepicker that can be used to override default locale code. */
+export const MAT_DATE_LOCALE = new InjectionToken<string>('MdDateLocale');
```

Comment:
I would just make the string value of the token the same as the name `'MAT_DATE_LOCALE'`